pkg/pillar/cmd/downloader/datastoreconfig.go

```diff
@@ -10,6 +10,7 @@ import (
 func handleDatastoreConfigCreate(ctxArg interface{}, key string,
 	configArg interface{}) {
 	handleDatastoreConfigImpl(ctxArg, key, configArg)
+	notifyResolveConfigByNewDatastoreConfig(ctxArg, key, configArg)
```

Comment:
Can you make this more like the existing 	checkAndUpdateDownloadableObjects(ctx, config.UUID)
meaning that is is called from the same place (handleDataStoreConfigImpl) and that the two functions have similar-looking names?

They implement the same concept (just on different objects) so no reason to have the reader think there are different concepts.